### ISSUE TITLE
error message

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#364]
when opening an imf file, sometimes this occurs

![image](https://user-images.githubusercontent.com/4609956/32949670-fc6e0e4c-cba3-11e7-9dee-01881a77d46f.png)

**Comment by gsautter** [comment]
While, ugly, this is not a problem ... This happens when GGI copies an IMF from the server to its local cache. For big IMFs, the connection can break up for network reasons. However, the IMF entries already downloaded stay in your local cache. Now if you try to open the same IMF again, the download of entries will start over right where it broke up. And even if you get this error on multiple successive attempts at opening a document from the server,it will succeed eventually, as download of the IMF entries to your local cache is differential and thus incremental - one of the reasons IMF is designed the way it is.
Bottom line: while ugly, this error will hardly ever actually block access to an IMF. You just have to try and open it multiple times before you get there.

**Comment by myrmoteras** [comment]
actually it is differrent. This occurred locally when I run the batch and then got this output. This has never been on TB. and the IMF files have been very small

**Comment by gsautter** [comment]
In that case, writing the IMF failed for some reason, similarly resulting in a broken ZIP (the server to GGI transfer also zips the IMF entries for efficiency).
To resolve this, simply delete the (most likely) broken IMF and re-run the batch. Due to the recently introduced storing of intermediate results (provided you specify the `CACHE` parameter as I advised), the IMF should pretty much readily pack from the cached data if you didn't clean that up manually. It will at most re-run the last batch processing step, i.e., the materials citations.
